internal/oci/verifier.go

```diff
@@ -34,13 +34,18 @@ import (
 	"github.com/sigstore/sigstore/pkg/signature"
 )
 
+// Verifier is an interface for verifying the authenticity of an OCI image.
+type Verifier interface {
+	Verify(ctx context.Context, ref name.Reference) error
+}
+
 // options is a struct that holds options for verifier.
 type options struct {
 	PublicKey []byte
 	ROpt      []remote.Option
 }
 
-// Options is a function that configures the options applied to a Verifier.
+// Options is a function that configures the options applied to a CosignVerifier.
```

Comment:
These options aren't specific to CosignVerifier, otherwise the func could just be `func(v *CosignVerifier)`. I think the original comment was correct.